lib/iris/fileformats/netcdf/saver.py

```diff
@@ -2210,6 +2213,34 @@ def add_ellipsoid(ellipsoid):
                     )
                     cf_var_grid.sweep_angle_axis = cs.sweep_angle_axis
 
+                # oblique mercator (and rotated variant)
+                # Checking grid_mapping_name instead of using isinstance()
+                #  since subclasses (i.e. RotatedMercator) upset mock tests.
```

Comment:
I should technically have called this "duck typing". Can update the comment if it is considered important.